plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -220,7 +220,6 @@ class ServicesTable extends React.Component {
   hasWebUI(service) {
     return (
       service instanceof Service &&
-      !isSDKService(service) &&
```

Comment:
Assumption: this check is moved into `Framework.js`, because SDK packages _always_ are `Framework`.